code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -194,270 +266,274 @@
 		ui.open()
 
 /obj/machinery/chem_master/ui_static_data(mob/user)
-	var/list/data = list()
-	data["categories"] = list()
+	. = list()
```

Comment:
Why are you changing this to the dot operator? I think I mentioned this in the last PR of your's but according to our style guide you should avoid using `.` like this unless it's for runtime safety. 
`var/list/data = list()` is fine.

Reply:
Resolved without answer? orrr

Reply:
Somehow I missed this. My bad